js/components/modalOverlay.js

```diff
@@ -18,10 +18,19 @@ class ModalOverlay extends ImmutableComponent {
       close = <button type='button' className='close pull-right' onClick={this.props.onHide}><span>&times;</span></button>
       title = <div className='sectionTitle' data-l10n-id={this.props.title} />
     }
-    return <div className='dialog'>
-      {close}
-      {title}
-      {this.props.content}
+    let customTitleClassesStr = (this.props.customTitleClasses ? this.props.customTitleClasses : '')
+
+    return <div className={'dialog ' + customTitleClassesStr}>
```

Comment:
something in this ModalOverlay change is breaking the `Buy with Coinbase` modal overlay:
![screen shot 2016-08-26 at 4 29 46 pm](https://cloud.githubusercontent.com/assets/549654/18023153/6bd7051e-6baa-11e6-8fc7-4ced55e1fdca.png)